NoDivide/astrum
# Fixed content jumping after focusing on sidebar

#### What does this PR cover?
Little fix for content jumping after hovering on application sidebar.

#### How can this be tested?
You can hover on sidebar and see there is no more content jump. Scrolling on sidebar and main content still works good.

---

#### Reviewers

**Review 1**

**Review 2** _(optional)_


By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.

---
